__tests__/keywords.ts

```diff
@@ -363,3 +358,49 @@ describe('mutation setKeywordAsSynonym', () => {
     expect(postKeywords).toMatchSnapshot();
   });
 });
+
+describe('flags field', () => {
```

Comment:
Maybe just add a null test to be safe so keyword not found

Reply:
There is already `should return null when keyword does not exist` test

Reply:
Ah ok missed it :)